update.sh

```diff
@@ -58,8 +57,8 @@ for version in "${versions[@]}"; do
 			.[$version].source[]
 			| select(.filename | endswith(".xz"))
 			|
-				"https://www.php.net/get/" + .filename + "/from/this/mirror",
-				"https://www.php.net/get/" + .filename + ".asc/from/this/mirror",
+				"https://www.php.net/distributions/" + .filename,
+				"https://www.php.net/distributions/" + .filename + ".asc",
```

Comment:
We should probably pull this URL change into a separate PR (although I don't have time right at this moment).

For context, these are the URLs listed on https://www.php.net/downloads (and the URLs Homebrew is using), so they seem more appropriate/correct than these older URLs we've been using.